redux/main/selectors.ts

```
@@ -168,7 +139,7 @@ export const selectComparison = (num) =>
 
     const arr = Object.values(tR);
     const ids = makeNewIdArr(arr.length);
-    arr.forEach((ob, idx) => (ob.id = ids[idx]));
+    arr.forEach((object, idx) => (object.id = ids[idx]));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)